test/SlowTests/Client/Attachments/AttachmentsReplication.cs

```diff
@@ -614,8 +614,7 @@ public async Task AttachmentsRevisionsReplicationAfterEnable2(Options options)
                 await SetupAttachmentReplicationAsync(store1, store2);
                 await SetupAttachmentReplicationAsync(store2, store1);
 
-                await store1.Operations.SendAsync(new EnforceRevisionsConfigurationOperation());
-                await store2.Operations.SendAsync(new EnforceRevisionsConfigurationOperation());
+                WaitForDocumentWithAttachmentToReplicate<User>(store2, "users/1", "foo", 15_000);
 
```

Comment:
why this was removed?

Reply:
because I think this is redundant at this point (we have less than 4 revisions at this point, so no need to trigger revisions configuration)

Reply:
ok